zoo/policies/competition_agent.py

```diff
@@ -0,0 +1,29 @@
+from distutils.command.config import config
+import subprocess
+import sys
+import os
+
+from pathlib import Path
+from smarts.core.agent import Agent
+
+
+class CompetitionAgent(Agent):
+    def __init__(self, policy_path):
+        root_path = Path(__file__).absolute().parents[2]
+        req_file = os.path.join(
+            root_path, "competition/track1/submission/requirements.txt"
+        )
```

Comment:
We should use the configuration provided.
```suggestion
        req_file = os.path.join(policy_path, "requirements.txt")
```